crates/bevy_render/src/primitives/mod.rs

```diff
@@ -95,6 +95,24 @@ impl Aabb {
     pub fn max(&self) -> Vec3A {
         self.center + self.half_extents
     }
+
+    /// Check if the point is at the front side of the plane.
+    /// Referenced from: [AABB Plane intersection](https://gdbooks.gitbooks.io/3dcollisions/content/Chapter2/static_aabb_plane.html)
+    #[inline]
+    pub fn is_forward_plane(
```

Comment:
I think that a better name could be `is_in_front_of_plane` or `is_in_halfspace`.